examples/chip-tool/commands/common/CHIPCommand.cpp

```diff
@@ -85,6 +88,26 @@ CHIP_ERROR GetAttestationTrustStore(const char * paaTrustStorePath, const chip::
     return CHIP_NO_ERROR;
 }
 
+CHIP_ERROR GetRevocationSet(const char * revocationSetPath, const chip::Credentials::RevocationSet ** revocationSet)
+{
+    revocationSetPath = revocationSetPath ? revocationSetPath : getenv(kRevocationSetPathVariable);
+
+    if (revocationSetPath == nullptr)
+    {
+        ChipLogError(chipTool, "No Revocation Sets provided.");
```

Comment:
But then we go ahead and pass null to the JSONFileRevocationSet constructor?

Reply:
This change has been removed from the PR.